src/main/java/org/opensearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -128,25 +124,22 @@ protected AbstractApiAction(
     protected void handleApiRequest(final RestChannel channel, final RestRequest request, final Client client) throws IOException {
 
         try {
-            // validate additional settings, if any
-            AbstractConfigurationValidator validator = getValidator(request, request.content());
-            if (!validator.validate()) {
-                request.params().clear();
-                badRequestResponse(channel, validator);
-                return;
-            }
             switch (request.method()) {
                 case DELETE:
-                    handleDelete(channel, request, client, validator.getContentAsNode());
+                    handleDelete(channel, request, client, null);
```

Comment:
Do `handleDelete` and `handleGet` use the JsonNode parameter anywhere?

Reply:
They are not, I intentionally left them as is, the next PR will change this logic